src/cli.rs

```diff
@@ -50,7 +50,7 @@ impl FromStr for Bootloader {
 #[command(args_conflicts_with_subcommands = true)]
 #[command(flatten_help = true)]
 pub struct ProveArgs {
-    #[clap(long = "with-bootloader", default_value_t = false)]
+    #[clap(long = "with-bootloader", default_value_t = true)]
```

Comment:
Geoffrey is implementing this + adding tests. With this change you do not provide a way to disable the bootloader mode. See #18 for the suggested implementation.